src/Plugin.php

```diff
@@ -237,13 +244,21 @@ public function saveOptionsHook(): void
     {
         $screen = \get_current_screen();
         if (false !== strpos($screen->id, 'towa-gdpr-plugin')) {
-            if (!isset($_POST['acf']['towa_gdpr_settings_hash']) || '' === $_POST['acf']['towa_gdpr_settings_hash'] || isset($_POST['save_and_hash'])) {
+            if (
```

Comment:
would extract this to a helper function so it becomes more readable.

Reply:
done, please recheck

Reply:
@volkanakb where? 😅